packages/vuelidate/src/index.js

```diff
@@ -40,16 +39,14 @@ export default function useVuelidate (validationsArg, state, registerAs) {
     injectToParent(validationResults, registerAs)
   }
 
-  return computed(() => {
-    if (registerAs && !childResultsKeys.value.length) {
-      return validationResults
-    } else {
-      return {
-        ...validationResults,
-        ...childResults.value
-      }
-    }
-  })
+  if (registerAs && childResultsKeys.value.length) {
+    return reactive({
+      ...validationResults,
+      ...childResults
+    })
+  } else {
+    return validationResults
```

Comment:
Why do we use `reactive` on the previous return statement but not here?

Reply:
I thought it would make things more consistent, as the `else` returns a reactive object too.

Reply:
oh ok! We reactive it somewhere else. Makes sense!